tests/raftstore/test_lease_read.rs

```diff
@@ -107,21 +127,20 @@ fn test_renew_lease<T: Simulator>(cluster: &mut Cluster<T>) {
     cluster.cfg.raft_store.raft_log_gc_threshold = 100;
```

Comment:
Those comments in this file need to be updated, since a "consistent read" has change from "write a raft log to raft quorum" to "handle a read index request by cluster with `ReadOnlyOption::Safe` option enabled".